src/vs/workbench/parts/files/electron-browser/views/explorerView.ts

```diff
@@ -471,34 +472,69 @@ export class ExplorerView extends TreeViewsViewletPanel implements IExplorerView
 
 		// Add
 		if (e.operation === FileOperation.CREATE || e.operation === FileOperation.COPY) {
-			const addedElement = e.target;
-			const parentResource = resources.dirname(addedElement.resource);
-			const parents = this.model.findAll(parentResource);
-
-			if (parents.length) {
+			const addedElement: IFileStat = e.target;
+
+			// compute the parent folder URI in which the new elements have been created
```

Comment:
The code for finding the parent folder could be removed if we would refactor runAction to also provide the parent folder. This would also require a change to the FileOperationEvent.